cartographer_grpc/framework/testing/rpc_handler_test_server.h

```diff
@@ -87,6 +87,10 @@ class RpcHandlerTestServer : public Server {
     WaitForHandlerCompletion(RpcHandlerWrapper<RpcHandlerType>::ON_FINISH);
   }
 
+  const typename RpcHandlerType::ResponseType &response() {
```

Comment:
Out of curiosity: It should compile without "typename" everywhere, shouldn't it?